### ISSUE TITLE
When Befriend Animal wears off , animal will still attack the opponent it was on..

### ISSUE THREAD

**Issue by noudess** [EQEmu/Server#674]
When Befriend Animal wears off , animal will still attack the opponent it was on, if the druid is out of reach.  I did this with sharks and got out of the water when it wore off.  The no longer befriended shark went back to eating his neighbor shark.

Assuming this is a removal from hate list(s) thing?

**Comment by mackal** [comment]
did some quick testing on live, appears they should be clearing their aggro list, and being removed from all others, then attacking the charmer. Will play more, need to find something that won't kill everything.

**Comment by noudess** [comment]
Think this should be closed @mackal or maybe I need to test on live in water?

**Comment by mackal** [comment]
No, I think there is something not working right, just haven't figured it out.

**Comment by noudess** [comment]
Ok, thx.

**Comment by mackal** [comment]
I'm also not sure it's working correctly. I know it does do a kick on live.

**Comment by Akkadius** [comment]
@noudess is this still a thing
